src/components/Grid/Grid.stories.tsx

```diff
@@ -24,7 +24,7 @@ interface Props {
 const Grid = ({ columnCount, rowCount, focus: focusProp, ...props }: Props) => {
   const [focus, setFocus] = useState(focusProp);
   const [columnWidth, setColumnWidth] = useState<Array<number>>(
```

Comment:
the focus state won't change every time the focusProp changes. We should have it either controlled or uncontrolled. this is a mix

Reply:
Fixed it 
If the focusProp is passed it will be respected instead of the inside one